edxval/api.py

```diff
@@ -373,8 +391,9 @@ def remove_video_for_course(course_id, edx_video_id):
 
 def get_videos_for_ids(
         edx_video_ids,
-        sort_field=None,
-        sort_dir=SortDirection.asc
+        sort_field=VideoSortField.created,
+        sort_dir=SortDirection.asc,
```

Comment:
I won't make such changes since it'll affect the sorting and direction even without pagination.